app/templates/components/cards/card-latest-jobs.hbs

```diff
@@ -4,8 +4,8 @@
       <h3 class="title">{{t 'latestUpdatedJobs'}}</h3>
     </div>
   </div>
-  <div class="card-block" id="large-table-jobs">
-    <table class="table table-hover">
+  <div class="card-block" id="large-table-jobs" >
+    <table class="table table-hover" >
```

Comment:
@yuricarvalhop, try to avoid this superfluous changes whichever only pollutes git history.